src/bindings/jna/libelektra5j/src/test/java/org/libelektra/exception/model/WarningEntryTest.java

```diff
@@ -19,7 +19,7 @@
 		String module = "kdb";
 		String reason = "This is a testerror";
 
-		final String warningKeyName = String.format ("warnings/#0", 0);
+//		final String warningKeyName = String.format ("warnings/#0", 0);
```

Comment:
Why did you comment out this line? If it isn't needed, we should just remove it.

Reply:
I figured, since the tests need some work anyways, i defer deleting it to a later point in time.
But i see your point and removed it.